message_detail_view.rb

```diff
@@ -63,11 +63,13 @@ def show_message(message, token, force=false)
   end
 
   def get_style_provider
-    color = UserConfig[:mumble_basic_bg]
+    fgcolor = UserConfig[:twitter_tweet_basic_color]
+    bgcolor = UserConfig[:twitter_tweet_basic_bg]
     Gtk::CssProvider.new.tap do |provider|
       provider.load_from_data(<<~CSS)
         *, *:active, *:disabled, *:hover, *:focus {
-          background-color: rgb(#{color[0] / 256}, #{color[1] / 256}, #{color[2] / 256});
+          color: rgb(#{fgcolor[0] / 256}, #{fgcolor[1] / 256}, #{fgcolor[2] / 256});
+          background-color: rgb(#{bgcolor[0] / 256}, #{bgcolor[1] / 256}, #{bgcolor[2] / 256});
```

Comment:
```suggestion
          color: #{css_rgb(fgcolor)};
          background-color: #{css_rgb(bgcolor)};
```

変換形式は同じなので、メソッドに切り出したほうが見やすそうです。
メソッドはこんな実装で。

```ruby
def css_rgb(uccolor)
  sprintf('#%02x%02x%02x', *uccolor.map { _1 >> 8 & 0xff })
end
```

irbにて確認
```
> css_rgb([0xffff, 0xcccc, 0x2323])
=> "#ffcc23"
```

UserConfigには、例えばRの強さに `0x80` を指定したい場合、8bit左シフトした結果とORを取って、 `0x8080` のようにして格納しているので、どちらか8bitを見ればOKです。